library/std/intrinsic.qs

```diff
@@ -396,6 +377,16 @@ namespace Microsoft.Quantum.Intrinsic {
     /// Qubit to which the gate should be applied.
     ///
     /// # Remarks
+    /// \begin{align}
+    ///     R_1(n, k) \mathrel{:=}
+    ///     \operatorname{diag}(1, e^{i \pi k / 2^n}).
+    /// \end{align}
+    ///
+    /// > [!WARNING]
+    /// > This operation uses the **opposite** sign convention from
+    /// > @"microsoft.quantum.intrinsic.r", and does not include the
+    /// > factor of $1/ 2$ included by @"microsoft.quantum.intrinsic.r1".
```

Comment:
I think it would be good to leave the warning in the summary so that it does show up in hover, since it alerts people to an important fact about how to use this operation.

Reply:
I've moved these to the summary sections as suggested, but I'm not sure about leaving in the `@` link syntax. It doesn't create links in the hover cards and may be confusing to users. Does the syntax create links in some other context?

Reply:
I've removed the `@` syntax.

Reply:
Sounds good, thanks!